apps/web/pages/api/availability/calendar.ts

```diff
@@ -76,7 +87,10 @@ async function getHandler(req: CustomNextApiRequest) {
   // get user's credentials + their connected integrations
   const calendarCredentials = getCalendarCredentials(user.credentials);
   // get all the connected integrations' calendars (from third party)
-  const { connectedCalendars } = await getConnectedCalendars(calendarCredentials, user.selectedCalendars);
+  const { connectedCalendars } = await getConnectedCalendars(
+    calendarCredentials,
+    user.userLevelSelectedCalendars
```

Comment:
Use `userLevelSelectedCalendars` only as this GET method isn't used in the app and it ensures the old behaviour